docs/articles/intro/getting-started/tutorial-1.md

```diff
@@ -3,53 +3,7 @@ uid: tutorial-1
 title: Part 1. Top-level Architecture
 ---
 
-# Part 1: Top-Level Architecture
-
-In this and the following chapters, we will build a sample Akka.NET application
```

Comment:
Why remove this?

Reply:
Ah I see, you moved it to a different page. So why should we still keep this page then?

Reply:
Got it - just moved the top-level overview of the tutorial.